### PR TITLE
Stage 2 before fix alexandra

### PR DESCRIPTION
**Pull request by malexa21**

### CODE REVIEW

File: src/views/CaloriesView.jsx [hunk 1]
```diff
@@ -22,45 +22,43 @@ import { v4 as uuidv4 } from "uuid";
...
     CALORIES_TARGET
   );
 
-  const percentage = CALORIES_TARGET * 0 + 75;
+  const percentage = CALORIES_TARGET;
 
   const formatPercentage = () => {
+    const roundedCalories = Math.floor(currentCalories);
     return (
+      <div>
```
Comment (AndreeaMereu) [inline src/views/CaloriesView.jsx:51]: I think that there is not need for this div. It is required if we have two or more components.

File: src/views/CaloriesView.jsx [hunk 2]
```diff
@@ -81,11 +80,18 @@ export const CaloriesView = () => {
         if (data.length === 0) {
           messageInstance.destroy();
           messageInstance.open({
-            type: "error",
+            type: "succes",
             content: `Could not find ${foodInputValue}`,
             duration: 5,
```
Comment (AndreeaMereu) [inline src/views/CaloriesView.jsx]: Adding an item notification should be disappear after 2 seconds.

File: src/views/CaloriesView.jsx [hunk 3]
```diff
@@ -184,19 +196,20 @@ export const CaloriesView = () => {
...
-      </>
+      </div>
     ));
   };
 
   const renderLoggedFoods = () => {
-    return <Empty description={"No logged foods"} />;
-
+    if(loggedFoods.length===0){
+      return <Empty description={"No logged foods"} />;
```
Comment (AndreeaMereu) [inline src/views/CaloriesView.jsx:209]: Here we should have the following description {"No recent searches"}

**Comment by cezarmocanu** [comment]
Approved, but discarded in favor of margin the other pull request

**Pull request closed without merging**